pkg/util/ssh.go

```diff
@@ -199,10 +200,15 @@ func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer)
 		} else {
 			// Some other kind of error happened (e.g. an IOError); consider the
 			// SSH unsuccessful.
-			err = fmt.Errorf("failed running `%s` on %s@%s: '%v'", cmd, user, host, err)
+			err = fmt.Errorf("failed running `%s` on %s: '%v'", cmd, remote, err)
 		}
 	}
-	return bout.String(), berr.String(), code, err
+	stdout, stderr := bout.String(), berr.String()
+	glog.Infof("[%s] stdout:    %q", remote, stdout)
+	glog.Infof("[%s] stderr:    %q", remote, stderr)
+	glog.Infof("[%s] exit code: %d", remote, code)
+	glog.Infof("[%s] error:     %v", remote, err)
```

Comment:
Move the printing part (at least) to somewhere in the e2e code. You're welcome to make an e2e/util or e2e/ssh package for it to live in, the e2e package is getting bloated.